### DISCUSSION TITLE
Welcome to ucblogo-code Discussions!

### DISCUSSION THREAD

**Discussion by jrincayc** [jrincayc/ucblogo-code#73] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dmalec** [comment]
Hello! My name is Dan and my first introduction to Logo was on an Apple II computer back in the 80s. In the last year, I've watched the plotter community explore the `10 PRINT` aesthetic; this got me thinking about creating similar one line plots using turtle graphics in Logo. I was happy to find ucblogo under active development (and thus able to run on my current computers).

Specific to this community - I'm enjoying digging into some of how Logo is implemented as well as learning more of the language / environment itself 😃

**Reply by brianharvey** [reply]
You'll enjoy this collection of entries in a contest to make an interesting picture in no more than 15 words of Logo code:
http://www.mathcats.com/gallery/15wordcontest.html

**Reply by dmalec** [reply]
Thank you! Those are awesome and I had not seen or heard of them before 😃. I'm interested in revisiting the ones I did and seeing if I can stay within the 15 word limit.
